doc/sphinx/changes.rst

```diff
@@ -581,6 +581,10 @@ Other changes in 8.10+beta1
   values will now highlight the differences
   (`#8669 <https://github.com/coq/coq/pull/8669>`_, by Jim Fehrle).
 
+- Proof General can now display Coq-generated diffs between proof steps in color.
+  (`#10019 <https://github.com/coq/coq/pull/10019>`_ and (in Proof General)
+  `#421 <https://github.com/ProofGeneral/PG/pull/421>`_, by Jim Fehrle).
```

Comment:
Handling it that way seems strange since #10019 was in the beta, although https://github.com/ProofGeneral/PG/pull/421 wasn't done at beta time.  How does the new process handle corrections to the changelist info (e.g. a major omission or something got listed that didn't make the release)?

No problem moving it if you still think that's best.